### ISSUE TITLE
Adding Content to site

### ISSUE THREAD

**Issue by schwagdawg** [aeonix/aeonix.github.io#27]
Couple complaints about the site and content/announcements/tutorials/etc. Before I dive into it is the single page site as must or something we can work around?

**Comment by iamsmooth** [comment]
It doesn't need to be a single page at all.

**Comment by BigslimVdub** [comment]
Are you still interested in re-working the webpage to include multiple pages?

**Comment by schwagdawg** [comment]
Yes I can. Do you have something in mind? Haven't been keeping up with Aeon as I moved overseas and didn't ship my servers yet.

**Comment by BigslimVdub** [comment]
We have set up a community webpage here https://aeoncommunity.github.io

From our community GitHub repo here https://github.com/AEONCommunity

If you want to contribute, feel free to join the repo and head over to discord/irc to chat with many members about their ideas. Right now we are working on a translations project and it includes the main webpage as content.

**Comment by aeon1234** [comment]
I would love to help where I can if this repo gets some love from smooth again.

**Comment by iamsmooth** [comment]
> this repo gets some love from smooth again

Not sure what you are looking for. 

Submit PRs, if they are (preferably) community-reviewed are easily reviewable they get merged.

The outstanding PRs all have some issues and none of them are blocking what would be new PRs as far as I can tell (no major reorgs or such).

**Comment by aeon1234** [comment]
>The outstanding PRs all have some issues 

then could you please write down these issues in the corresponding PRs?
this would help to fix them and get them merged soon, at least I do not know why both of mine don't get merged and so I cannot fix them.

I also do not know how to organize a complete website overhaul. I think others also do not know. so maybe you could write something about that and how we get consensus. otherwise this could be a big time waste in worst case for everyone who is motivated to do and this is nothing we should aim for with our small community.

thank you.

**Comment by ivoryguru** [comment]
Hello I am proposing a new website to the community.  It is designed with the community in mind. It is not some bombastic sales pitch or complicated cryptographers diary. It is intended to be understandable by everyone, even your Mother. It is based on the original Aeon whitepaper and can be previewed at this domain. [aeon-crypto.org](aeon-crypto.org) . 

Links are dying everyday and the website becomes ugly and cluttered. We don't need to list exchanges trading aeon or block explorers, there are countless ways to find these resources. That is why this new version is intended to be a stable website only following the changes in the code. It can be easily translatable for many languages and can also allow offsource information to the community to provide alternate sources of AEON information on pages such as https://aeoncommunity.github.io/. I hope you will consider adding this as the primary AEON website. 

[aeon-crypto.org](aeon-crypto.org)
